### ISSUE TITLE
How to run Seldon Batch Processing without Kubeflow/Argo/Airflow is missing

### ISSUE THREAD

**Issue by SivaSankariRamamoorthy** [SeldonIO/seldon-core#4169]
## Describe the bug

So, I tried framing seldon-batch-processor command and ran it in debug mode. Got an exception without the exception reason. Sample command that I tried is as follows -

seldon-batch-processor -d <deploy-name>  -h <endpoint-of-service> -a raw -i input.data -o output.data -l debug --ssl-verify  false -n <namespace>

The result I received is
2022-06-21 12:46:13,005 - batch_processor.py:604 - ERROR:  Exception:
2022-06-21 12:46:13,105 - batch_processor.py:219 - INFO:  Total processed instances: 1

As you can see, the exception doesn't contain the reason for which error is thrown. Need help to understand how to run seldon batch processing without using workflow managers

## Model Details 
* Images of your model: [Output of: `kubectl get seldondeployment -n <yourmodelnamespace> <seldondepname> -o yaml | grep image:` where `<yourmodelnamespace>`]
* Logs of your model: [You can get the logs of your model by running `kubectl logs -n <yourmodelnamespace> <seldonpodname> <container>`]

**Comment by axsaucedo** [comment]
Can you share the content / structure of your input? Are you making sure it's the same structure as the one provided in the examples?

**Comment by SivaSankariRamamoorthy** [comment]
So, I have tried 2 ways of framing input.data. 

1. Tried the format given in the seldon doc as follows. But, where can I get the values to assign to 'meta' key? So, I left them blank and tried running the seldon batch processing command. Got Exception as 'expecting value line 2 column 1'

![image](https://user-images.githubusercontent.com/9278475/175919007-9cdb8335-4bfb-4ea2-a8e4-de5956cef1fb.png)

2. So, tried just by passing the input required for the app to run, something like this - {'inputKey': 'String to the service'}. On passing this plain JSON input, seldon is implicitly converting the input into the elements Got Exception with no error message
